pkg/trace/stats/concentrator.go

```diff
@@ -165,7 +165,8 @@ func (c *Concentrator) addNow(pt *traceutil.ProcessedTrace, containerID string)
 	}
 	for _, s := range pt.TraceChunk.Spans {
 		isTop := traceutil.HasTopLevel(s)
-		if !(isTop || traceutil.IsMeasured(s)) || traceutil.IsPartialSnapshot(s) {
+		isRemoteOutgoing := traceutil.IsRemoteOutgoing(s)
+		if !(isTop || traceutil.IsMeasured(s) || isRemoteOutgoing) || traceutil.IsPartialSnapshot(s) {
```

Comment:
Happy to split this up if the readability is getting dicey.

Reply:
I'm curious why not for all non-internal spans?